internal/connector/pingone/sso/resources/pingone_population_default_identity_provider.go

```diff
@@ -0,0 +1,113 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/patrickcping/pingone-go-sdk-v2/management"
+	"github.com/pingidentity/pingcli/internal/connector"
+	"github.com/pingidentity/pingcli/internal/connector/common"
+	"github.com/pingidentity/pingcli/internal/connector/pingone"
+	"github.com/pingidentity/pingcli/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingOnePopulationDefaultIdp{}
+)
+
+type PingOnePopulationDefaultIdp struct {
+	clientInfo *connector.PingOneClientInfo
+}
+
+// Utility method for creating a PingOnePopulationDefaultIdp
+func PopulationDefaultIdp(clientInfo *connector.PingOneClientInfo) *PingOnePopulationDefaultIdp {
+	return &PingOnePopulationDefaultIdp{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingOnePopulationDefaultIdp) ResourceType() string {
+	return "pingone_population_default_identity_provider"
+}
+
+func (r *PingOnePopulationDefaultIdp) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+	l.Debug().Msgf("Exporting all '%s' Resources...", r.ResourceType())
+
+	importBlocks := []connector.ImportBlock{}
+
+	populationData, err := r.getPopulationData()
+	if err != nil {
+		return nil, err
+	}
+
+	for populationId, populationName := range populationData {
+		populationDefaultIdp, err := r.getPopulationDefaultIdp(populationId)
+		if err != nil {
+			return nil, err
+		}
+		if populationDefaultIdp == nil {
+			continue
+		}
+
+		commentData := map[string]string{
+			"Export Environment ID":                r.clientInfo.ExportEnvironmentID,
+			"Population Default Identity Provider": *populationDefaultIdp,
+			"Population ID":                        populationId,
+			"Population Name":                      populationName,
+			"Resource Type":                        r.ResourceType(),
+		}
+
+		importBlock := connector.ImportBlock{
+			ResourceType:       r.ResourceType(),
+			ResourceName:       populationName,
```

Comment:
Differentiate the name from the `pingone_population` resource: `fmt.Sprintf("%s_default_identity_provider", populationName)`